app/main.py

```diff
@@ -16,4 +16,14 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+
+sorted_variables = {
+    "mutable": [],
+    "immutable": [],
+}
+
+sorted_variables["mutable"] = [marks, collection_of_coins, my_favourite_films]
+sorted_variables["immutable"] = [lucky_number, pi, one_is_a_prime_number,
+                                 name, profile_info]
+
+print(sorted_variables)
```

Comment:
it is better not to create variables if they are used only once

```suggestion
sorted_variables = {
    "mutable": [marks, collection_of_coins, my_favourite_films],
    "immutable": [lucky_number, pi, one_is_a_prime_number, name, profile_info],
}

```